src/_posts/platform/getting-started/2000-01-01-how-to-migrate-from-heroku.md

```diff
@@ -1,47 +0,0 @@
----
```

Comment:
This page is called from two places:
https://github.com/Scalingo/documentation/blob/5ac1f04e478079b4bdddfd60dff827beb546d9fd/_config.yml#L55

and

https://github.com/Scalingo/documentation/blob/5ac1f04e478079b4bdddfd60dff827beb546d9fd/redirections.yml#L161-L163

The redirection can be deleted. For the `_config.yml`, we may want to ask the marketing as it's currently highlighted on the doc index:

![2022-05-02_19-50](https://user-images.githubusercontent.com/1567651/166298790-de9cc61f-46dd-4911-8f76-98efd1dae461.png)

Can you ask marketing what is their thought about this?

Moreover, deleting this page means we should clearly deprecate this tool (https://github.com/Scalingo/heroku2scalingo). You should open a PR to mark it as deprecated, then ask Léo to archive the repository. You can take inspiration from this PR: https://github.com/Scalingo/puppeteer-buildpack/pull/1